app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/vaultunlock/VaultUnlockScreen.kt

```diff
@@ -108,6 +113,17 @@ fun VaultUnlockScreen(
                     cipher = event.cipher,
                 )
             }
+
+            VaultUnlockEvent.Fido2CredentialAssertionError -> {
+                fido2CompletionManager.completeFido2Assertion(
+                    result = Fido2CredentialAssertionResult.Error,
+                )
+            }
+            VaultUnlockEvent.Fido2GetCredentialsError -> {
+                fido2CompletionManager.completeFido2GetCredentialRequest(
+                    result = Fido2GetCredentialsResult.Error,
+                )
+            }
```

Comment:
Do we need to set the `title = dialog.title` below?

Reply:
Yes! Thank you.